.gitignore

```diff
@@ -4,3 +4,4 @@
 /cover
 erl_crash.dump
 *.ez
+.elixir_ls
```

Comment:
You probably want this in you global `.gitignore`

Reply:
TIL you can have a global `.gitignore`.